docs/user-guide/self_rewarding.rst

```diff
@@ -0,0 +1,214 @@
+.. include:: /content/nemo.rsts
+
+Model Alignment by Self-Rewarding Language Models
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+
+Original paper: https://arxiv.org/abs/2401.10020
+Meta Self-Rewarding paper: https://arxiv.org/abs/2407.19594
+
+The NeMo framework supports efficient model alignment via the NeMo Aligner codebase.
+
+All algorithms in NeMo Aligner will work with any GPT based model that is from mcore(i.e in the config it has ``mcore_gpt=True``). For the purposes of this tutorial, we will go through the entire Self-Rewarding pipeline using the newly released `2B GPT model with 4096 sequence length <https://huggingface.co/nvidia/GPT-2B-001>`__.  This same tutorial also works for other GPT models(such as LLaMa2) of any size.
+
+Obtaining a pretrained model
+############################
+To start, we must first get a pretrained model to align. There are 2 models we recommend to get started. The rest of the tutorial will work with either model, but for demonstration purposes we will use the smaller 2B model. 
```

Comment:
Spell out number "two" and fix punctuation.

To start, we must first get a pretrained model to align. There are two models we recommend to get started. The rest of the tutorial will work with either model, but for demonstration purposes, we will use the smaller 2B model.